src/components/QuoteBanner/QuoteBanner.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const QuoteBanner = ({children,className}) => {
+  return (
+    <div 
+    className={{
+        className
+    }}
+    style={{
```

Comment:
nit: now since we have moved these styles into the QuoteBanner and we also have className, we can actually 
 give this component its own independent className to and append the className from props to add a final className and then apply style to that class. Its just a further improvement we can look at, nothing rigid

Reply:
Will take it in follow up